docs/meshstack.aks-developer-platform-quickstart-guide.md

```diff
@@ -0,0 +1,299 @@
+---
+id: meshstack.aks-developer-platform-quickstart-guide
+title: AKS Developer Platform Guide
+---
+
+## Introduction
+
+Welcome to the **First Developer Platform guide**! This guide provides step-by-step instructions to help you set up your first developer platform experience incorporating AKS, GitHub & GitHub Actions to deliver an easy-to-use starter kit to your application teams.
+
+## Goals
+
+By the end of this guide, you will have built a complete starter kit for your users that will provide:
+
+- A new GitHub repository
+- A “dev” project with:
+    - an AKS namespace
+    - a GitHub actions workflow with credentials to the AKS namespace
+- A “prod” project with:
+    - an AKS namespace
+    - a GitHub actions workflow with credentials to the AKS namespace
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+- You have a basic understanding of building blocks in meshStack.
+    - *If you don’t, we recommend running through the [AWS S3 guide](./meshstack.building-aws-quickstart-guide.md) to learn how building blocks work.*
+- You have integrated an AKS cluster into meshStack.
+    - *If you don’t, we recommend running through the [First Platform guide for AKS](./meshstack.aks-platform-quickstart-guide.md) where you are guided to integrate an AKS cluster as a platform in meshStack*
+
+## What will we do in this guide?
+
+Shortly put, we will set up the following in this guide:
+
+- A building block definition that allows application teams to provision new repositories.
+- A building block definition that adds a GitHub Action workflow with the right credentials in the AKS namespace so application teams have an out-of-the-box working Kubernetes CI/CD.
+- A building block definition that combines all these elements & more into a single packaged starter kit.
+
+## 1. GitHub Repository
+
+### Preparation
+
+The GitHub Repository building block definition needs access to your GitHub organization.
+Follow the following steps to create a GitHub App that will be used to manage the repositories.
+
+1. [**Register a GitHub App**](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app)
+    1. Make sure you give it sufficient permissions. When creating the app you’ll see a section “Repository permissions”. Under here make sure that you set the following:
```

Comment:
The numbering looks very weird for this part. Can you instead go for bullet points starting on the second hierarchy?

Reply:
It actually renders differently: 
<img width="817" alt="image" src="https://github.com/user-attachments/assets/68509c38-5feb-4ded-81bd-515587f9dfe3" />